src/Microsoft.NetCore.Analyzers/Core/Security/DoNotAddSchemaByURL.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using Analyzer.Utilities;
+using Analyzer.Utilities.Extensions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.NetCore.Analyzers.Security
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public sealed class DoNotAddSchemaByURL : DiagnosticAnalyzer
+    {
+        internal const string DiagnosticId = "CA5366";
```

Comment:
Can you call this one "CA3061"?  There used to be an FxCop rule that was trying to do the same thing.